models/aether_lat-lon/matlab/new_varname_file.m

```diff
@@ -0,0 +1,155 @@
+function new_varname_file(data_dir, member, nblocks)
+
+% Converts Aether restart file names to updated (2024-1-17) versions.
```

Comment:
There is no mention of this matlab file 'new_varname_file.m' in the documentation.  Is this something that you would expect people to use?  If not, then I'd remove it from this pull request.

Reply:
I don't expect people to use it as a regular part of assimilating.
But if Aether variable names change again in the future, and still use difficult characters 
(blanks, \s, etc.) this file would be very useful for transforming old files into new ones.
I was not able to make NCO or fortran do it.  Python might be able to, but that hasn't been written
(? @johnsonbk).
Given that, I'll either write documentation or remove it from the PR.  Which do you prefer?

Reply:
I'd say a line or two in the documentation saying new_varname_file as a tool